src/System.Private.Xml/tests/XmlSerializer/XmlSerializerTests.cs

```diff
@@ -117,6 +117,7 @@ public static void Xml_DateTimeAsRoot()
     }
 
     [Fact]
+    [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "XmlCustomFormatter.FromTime is quirked, which causes behavior difference. See the method for more details.")]
```

Comment:
Same comment as @danmosemsft maybe we should test the quirked behavior using the switch value or parsing the framework version out of `AppDomain.CurrentDomain.SetupInformation.TargetFrameworkName`. Because in the future when we get to have a greater TFM then this would be disabled.